src/sentry/tasks/assemble.py

```diff
@@ -231,10 +240,81 @@ def _store_single_files(archive: ReleaseArchive, meta: dict, count_as_artifacts:
             _upsert_release_file(file, None, _simple_update, kwargs, extra_fields)
 
 
+def _normalize_headers(headers: dict) -> dict:
+    return {k.lower(): v for k, v in headers.items()}
+
+
+def _normalize_debug_id(debug_id: Optional[str]) -> Optional[str]:
+    try:
+        return normalize_debug_id(debug_id)
+    except SymbolicError:
+        return None
+
+
+def _extract_information_from_manifest(manifest: dict) -> List[Tuple[SourceFileType, str]]:
+    debug_ids_with_types = []
+
+    files = manifest.get("files", {})
+    for filename, info in files.items():
+        headers = _normalize_headers(info.get("headers", {}))
+        debug_id = _normalize_debug_id(headers.get("debug-id", None))
```

Comment:
I would explicitly check if the `debug-id` is not `None` before calling into `normalize_debug_id` and swallowing the error there.